### ISSUE TITLE
15KHz 240p widescreen output and unable to load profile via remote preset

### ISSUE THREAD

**Issue by MarcoRetro** [marqs85/DExx-vd_isl#5]
Great product. I’m wondering if there’ll be a 240p 15KHz WS mode, as currently 16:9 content is further squished. Also, is there a way to load profiles with the remote? The bottom left profile button doesn’t seem to do anything

**Comment by marqs85** [comment]
So basically you'd like to have a "240p WS" scaler CRT output preset? It's certainly trivial to add even though I can't think of much 16:9 content that I'd run at 240p. Handler for profile load button is not yet implemented but it's not much work and could be included in next firmware as well.

**Comment by MarcoRetro** [comment]
Yes correct, 240p WS. It’s a niche user-case but for the few of us downscaling to a widescreen 15KHz CRT. Thanks Marqs!
